apps/src/p5lab/poetry/PoetryLibrary.js

```diff
@@ -448,6 +459,24 @@ export default class PoetryLibrary extends CoreLibrary {
     }
   }
 
+  drawTextHighlight(item) {
+    const padding = 2;
+    const w = drawUtils.getTextWidth(this.p5, item.text, item.size);
```

Comment:
nit: prefer `width` and `height` over `w` and `h`

Reply:
updated! https://github.com/code-dot-org/code-dot-org/pull/43421/commits/d262fe2f09f4913d32ce1ca0ddb4a6283e9bc7c7